code/pom.xml

```diff
@@ -81,6 +76,53 @@
         </mailingList>
     </mailingLists>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <junit-version>4.12</junit-version>
+        <maven-plugin.version>1.0.0</maven-plugin.version>
+        <findbugs.exclude>${project.basedir}/exclude.xml</findbugs.exclude>
+        <findbugs.skip>false</findbugs.skip>
+        <findbugs.threshold>Low</findbugs.threshold>
+        <legal.doc.source>${project.basedir}/..</legal.doc.source>
+        <vendor.name>Oracle Corporation</vendor.name>
+    </properties>
+
+    <modules>
+        <module>fastinfoset</module>
+        <module>roundtrip-tests</module>
+        <module>utilities</module>
+        <module>samples</module>
+    </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.maven</groupId>
+                <artifactId>maven-plugin-tools-api</artifactId>
+                <version>2.1</version>
+            </dependency>
+            <dependency>
+                <groupId>com.sun.xsom</groupId>
+                <artifactId>xsom</artifactId>
+                <version>20140925</version>
```

Comment:
this one should have the latest version in jaxb-ri repo :-/

Reply:
This shall wait for JAXB RI staging release.

Reply:
ok